test/tools/libtesteth/ImportTest.cpp

```diff
@@ -205,6 +205,68 @@ set<eth::Network> ImportTest::getAllNetworksFromExpectSections(
     return networkSet;
 }
 
+void ImportTest::parseJsonStrValueIntoSet(json_spirit::mValue const& _json, set<string>& _out)
+{
+    if (_json.type() == json_spirit::array_type)
+    {
+        for (auto const& val : _json.get_array())
+            _out.emplace(val.get_str());
+    }
+    else
+        _out.emplace(_json.get_str());
+}
+
+vector<int> parseJsonIntValueIntoVector(json_spirit::mValue const& _json)
```

Comment:
Either put it into unnamed namespace or make it private static member of the class